pkg/apis/core/types_shoot.go

```diff
@@ -154,6 +154,18 @@ type ShootStatus struct {
 	MigrationStartTime *metav1.Time
 	// Credentials contains information about the shoot credentials.
 	Credentials *ShootCredentials
+	// LastMaintenance holds information about the last maintenance operations on the Shoot.
+	LastMaintenance *LastMaintenance
```

Comment:
Please enhance https://github.com/gardener/gardener/blob/master/docs/usage/shoot_maintenance.md with information about this new field.

Reply:
I guess this is not the right place this is more about how/what can be configured in maintenance.

Reply:
I can't follow. This document explains everything about maintenance for users. Please add some explanation that the `Shoot` status contains information about the last operation.